### PR TITLE
backend: comply with new golangci tests

### PR DESCRIPTION
**Pull request by Beerosagos**
Upgrading `golanci-lint` to v1.50 showed several nits in the code. With this I updated the code where possible and skipped three new tests.

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -131,6 +131,9 @@ linters:
     - makezero
     - nolintlint
     - gocyclo
+    - nosnakecase
```
Comment (Beerosagos) [inline .golangci.yml:134]: This generates errors like 
```
frontends/qt/server/server.go:85:26: _Ctype_int contains underscore. You should use mixedCap or MixedCap. (nosnakecase)
func backendCall(queryID C.int, s *C.char) {
```

File: .golangci.yml [hunk 2]
```diff
@@ -131,6 +131,9 @@ linters:
...
+    - interfacebloat
```
Comment (Beerosagos) [inline .golangci.yml:135]: This generates errors like this. There are a lot of them
```
backend/handlers/handlers.go:66:14: the interface has more than 10 methods: 37 (interfacebloat)
type Backend interface {
```

File: .golangci.yml [hunk 3]
```diff
@@ -131,6 +131,9 @@ linters:
...
+    - revive
```
Comment (Beerosagos) [inline .golangci.yml:136]: This generates errors like this: we could also add package comments, but I don't know if it is really useful and I think that we are missing a lot of them.
```
util/observable/action/action.go:15:1: package-comments: should have a package comment (revive)
package action
```

**Review by benma — APPROVED** [review]
nice!

**Pull request merged**